randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -10,12 +10,13 @@ Extra - asset_id: collision_camera_000
   * Extra - start_point_actor_name: ST_SaveStation_004
   > Door to Surface Stash
       Trivial
-  > Top of Mountain
-      Climb Rooms Vertically (No High Jump)
-  > Dock to Surface Crumble Block Challenge (Top)
-      Climb Rooms Vertically (No High Jump) and Lay Power Bomb
   > Dock to Surface - West
       Baby Metroid
+  > Top Left Midair
```

Comment:
Seems odd to have a random node in the air vs just connecting to an existing node but I guess this works out better?

Reply:
This makes it nicer to connect to the other nodes, yes. AM2R and probably a few other games use it too in some places.
it's basically short for "if you can reach the top, you can reach every other connection from there too"